### DISCUSSION TITLE
Edited Community Plays got Reset by SecOps Automation

### DISCUSSION THREAD

**Discussion by NexusChak** [Security-Onion-Solutions/securityonion#11754] [Unsupported Versions]
SO Version; 2.4.20

I'm wondering if there is such a mechanism that SecOps Automation account in playbook would reset plays after a certain period of time?

**Comment by defensivedepth** [comment]
Plays imported from the Sigma community repo managed by Playbook and updates from that repo are pulled down on a daily basis.
